bluetoothServicesComponent/modem_info_service.c

```diff
@@ -15,59 +15,67 @@
 #include "modem_info_service.h"
 #include "org.bluez.GattCharacteristic1.h"
 #include "org.bluez.GattService1.h"
+#include "org.bluez.GattDescriptor1.h"
 
 #define MODEM_INFO_FSN_CHARACTERISTIC_UUID "2A25"
-#define MODEM_INFO_IMEI_CHARACTERISTIC_UUID "2A27"
+#define MODEM_INFO_IMEI_CHARACTERISTIC_UUID "865d"
```

Comment:
I think I wasn't clear enough in my previous comment.  16-bit and 32-bit UUIDs are reserved for standardized use by the Bluetooth SIG.  You should assign a full 128-bit (16-byte) uuid.  Use this one which I just randomly generated:
`fb22d0b6-7c72-4e29-a156-df6518f69ec4`